### ISSUE TITLE
When training the val loss =nan always. 

### ISSUE THREAD

**Issue by qingchengboy** [qqwweee/keras-yolo3#70]
hi, thanks for u code. When i train my data, just **one** class,  i have already changed the filters=3*(class num+5)=18 in yolov3.cfg file.  But the loss value is around 27 and does not continue to fall. The val loss always =nan, and the generated model does not detect the results.  i am confused --

**Comment by ewen1024** [comment]
I am in similar situation, though when my loss came to around, the next epoch might be nan for loss, than the program stopped

**Comment by elcronos** [comment]
Hey guys so just to clarify in your case, after how many epochs you start getting a loss different than nan?

I first tried and stopped after 5 epochs. Then I changed to 100. I just wanted to know how many epochs you will start getting some results !

<img width="1539" alt="screen shot 2018-06-01 at 2 08 37 pm" src="https://user-images.githubusercontent.com/9023043/40827380-01cdacf2-65b0-11e8-81c4-3edcc9ce5a03.png">

**Comment by thanif** [comment]
I was experiencing the same problem while using tiny-yolo and the reason was that the weights were wrong. I downloaded the yolov3-tiny.weights from darknet along with the yolov3-tiny.cfg and convert it using the command : 

python3 convert.py yolov3-tiny.cfg yolov3-tiny.weights tiny-yolo.h5

 The problem was resolved.
